particles/fcc/particle_link.py

```diff
@@ -0,0 +1,27 @@
+from heppy.particles.particle_link import Particle_Link as BaseLink
+from pod import POD
+from heppy.papas.data.identifier import Identifier
+
+from ROOT import TLorentzVector
+from ROOT import TVector3
+
+class Particle_Link(BaseLink, POD):
```

Comment:
I don't think this class is necessary.

Reply:
necessary, just to build the _objid for the incoming and outgoing in the constructor